### DISCUSSION TITLE
TypeError: concat() takes 1 positional argument but 2 were given

### DISCUSSION THREAD

**Discussion by RashedCh** [tbepler/topaz#180] [cryoSPARC]
Hello,
I'm running this issue with Topaz version 0.2.5a that was discussed previously in a cryoSPARC thread (https://discuss.cryosparc.com/t/topaz-train-typeerror-concat-takes-1-positional-argument-but-2-were-given/11240/3). There was really no solution other than downgrading to an earlier version of Topaz (0.2.4). Here's what I get (through both inside and outside of cryoSPARC):

$ bash /usr/local/anaconda3/envs/topaz/bin/topaz.sh train_test_split --number 89 --seed 472515481 --image-dir /XX/preprocessed /XX/topaz_particles_processed.txt
Splitting 445 micrographs with 23638 labeled particles into 356 train and 89 test micrographs
Traceback (most recent call last):
  File "/usr/local/anaconda3/envs/topaz/bin/topaz", line 33, in <module>
    sys.exit(load_entry_point('topaz-em==0.2.5', 'console_scripts', 'topaz')())
  File "/usr/local/anaconda3/envs/topaz/lib/python3.10/site-packages/topaz/main.py", line 148, in main
    args.func(args)
  File "/usr/local/anaconda3/envs/topaz/lib/python3.10/site-packages/topaz/commands/train_test_split.py", line 108, in main
    targets_train = pd.concat(groups_train, 0)
**TypeError: concat() takes 1 positional argument but 2 were given**

Can someone help please?

**Comment by DarnellGranberry** [comment]
I've updated that line to conform to pd.concat now only taking keyword arguments. The fix is available now on master, if you can install topaz from source; otherwise, it will be available in the next release.
